src/components/material/MaterialDescription.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { WorkMediumFragment } from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import HorizontalTermLine from "../horizontal-term-line/HorizontalTermLine";
+
+interface MaterialDescriptionProps {
+  work: WorkMediumFragment;
+  description?: string;
+}
+
+const MaterialDescription: React.FC<MaterialDescriptionProps> = ({
+  work,
+  description
+}) => {
+  const t = useText();
+
+  const seriesMembersList = work.seriesMembers.map(
```

Comment:
I think you should add a note that we are not sure but we assume that titles.main is the right place to find the title. We should do this globally becasuse we have looked it up in search result items as well